dsf/org.eclipse.cdt.dsf.ui/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-Vendor: %providerName
 Bundle-SymbolicName: org.eclipse.cdt.dsf.ui;singleton:=true
-Bundle-Version: 2.7.300.qualifier
+Bundle-Version: 2.7.400.qualifier
```

Comment:
As you are changing what the API class DsfSteppingModeTarget implements, you need to bump this bundle to 3.0.0 or keep the `implements ITargetProperties` and do one of these in DsfSteppingModeTarget:

1. Leave the implementation of the 2 functions empty.
2. Throw an exception in the 2 functions.
3. Glue the functions together with the new `fPreferences` object in some way.

Reply:
I think keeping the old implementation is good idea. Because it is also public API.